examples/wrapped/plotscripts/raw_out_testplots.py

```diff
@@ -51,7 +52,7 @@
 surface_pressure_plots = []
 surface_temperature_plots = []
 
-relative_errors = {}
+relative_errors: Dict[str, np.ndarray] = {}
```

Comment:
was this type-hint required? I thought adding type-hints was still optional, or did you just add it because you were looking at this script and added it?

Reply:
Type hints are required by mypy for dictionaries whose initialization does not indicate the types. If this were `relative_errors = {"foo": np.ndarray([1.0])}` mypy wouldn't ask for a type hint. This is only true about a couple built-in mutable classes, like `list` and `dict`.

If you don't type hint something, mypy secretly type hints it by seeing how you initialized it.

Reply:
seems overly restrictive to me, but we can follow it given we are mostly trying to push things into gt4py

Reply:
In practice this comes up rarely enough that I forget it's required. I think I've had to type hint things like this fewer than 10 times, ever.